index.js

```diff
@@ -3,16 +3,47 @@ const cards = document.querySelectorAll('.memory-card')
 const restartBtn = document.querySelector('#restart-btn')
 const timer = document.querySelector('#timer')
 let counter = 0;
+let memoryCounter = 5;
```

Comment:
values which convey special meaning are generally define as constant and in UPPERCASE

Reply:
memoryCounter is supposed to be a variable just like the counter and its changing value is used to change the inner HTML of timer id tag so it can't be a const